src/roi/getMask.ts

```diff
@@ -4,27 +4,22 @@ import { Roi } from './Roi';
 
 export interface GetMaskOptions {
   /**
-   * Should the inner borders be returned too?
+   * Should the ROI holes be filled in the resulting mask?
    *
    * @default false
    */
-  innerBorders?: boolean;
-  /**
-   * Consider pixels connected by corners? This option is only useful if filled = false.
-   *
-   * @default false
-   */
-  allowCorners?: boolean;
+  solidFill?: boolean;
 }
+
 /**
- * Generate a mask of an ROI. You can specify the kind of mask you want using the `kind` option.
+ * Generate a mask of an ROI.
  *
  * @param roi - The ROI to generate a mask for.
  * @param options - Get mask options.
  * @returns The ROI mask.
  */
 export function getMask(roi: Roi, options: GetMaskOptions = {}): Mask {
-  const { innerBorders = true } = options;
+  const { solidFill = true } = options;
```

Comment:
Should be false by default I think. And please check that both case are covered by the test cases.